pipeline-model-definition/src/test/java/org/jenkinsci/plugins/pipeline/modeldefinition/steps/DeclarativePropsStepTest.java

```diff
@@ -173,6 +173,7 @@ public void runsOnCorrectSlave() throws Exception {
         s.setLabelString("thisone");
         env(s).put("DOCKER_INDICATOR", "CORRECT").set();
         GlobalConfig.get().setDockerLabel("thisone");
+        GlobalConfig.get().setRegistry(null);
```

Comment:
Check that, it did need to be here. In fairness, I'm not entirely sure *why* it started failing here and not earlier, but...yeah.